src/ngx_http_vhost_traffic_status_shm.h

```diff
@@ -18,10 +18,12 @@ typedef struct {
     ngx_uint_t   filter_used_node;
 } ngx_http_vhost_traffic_status_shm_info_t;
 
+#define NGX_HTTP_VHOST_TRAFFIC_STATUS_STATUS_CODE_SLOT_UNDEFINED -1
```

Comment:
@wpjunior 

Instead of introduce new constant, should we use 0 as special code slot?
So that way we can represent about slot counter type as `ngx_uint_t`, we don't have to be redundant type casting, expect to improve readability code.

Moreover, this is one of my idea, it would be fine that slot outputs as `other` status which means it is not specified in conf.

Reply:
huuum @u5surf sounds great

Reply:
@u5surf read my last commit